src/Ticketing/TMS.Ticketing.Domain/Events/EventEntity.cs

```diff
@@ -0,0 +1,24 @@
+﻿using TMS.Ticketing.Domain.Common;
+
+namespace TMS.Ticketing.Domain.Events;
+
+public sealed class EventEntity : IEntity<Guid>
+{
+    public required Guid Id { get; init; }
+    
+    public required string Name { get; set; }
+
+    public List<KeyValePair>? Details { get; set; }
```

Comment:
wouldn't it be better to have here a Dictionary? KeyValue is very similar to a map and can be here two entries with the same Key?

Reply:
I want to use custom objects to avoid potential deserialization issues and be able to implement validations as I want. Regarding the same keys, I leave it up to the user.